qemu/tests/virtio_mem_numa_basic.py

```diff
@@ -0,0 +1,46 @@
+import time
+
+from virttest import error_context
+
+from virttest.utils_misc import normalize_data_size
+from provider import virtio_mem_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Memory share and discard-data hotplug test
+    1) Boot guest with two virtio-mem devices
+    2) Check virtio-mem devices
+    3) Resize virtio-mem devices
+    4) Check virtio-mem devices
+    5) Resize virtio-mem devices to the maximum
+    6) Check virtio-mem devices
+    7) Resize virtio-mem devices to the minimum
+    8) Check virtio-mem devices
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    timeout = int(params.get("login_timeout", 240))
+    threshold = params.get_numeric("threshold", target_type=float)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login(timeout=timeout)
+    for i in range(2):
+        requested_size_vmem = params.get("requested-size_test_vmem%d" % i)
+        device_id = params.get("memdev_memory_vmem%d" % i)
+        device_id = "virtio_%s" % device_id
```

Comment:
I would prefer to use `vm.devices.get_by_properties/get_by_params`, because we know the driver is `virtio-mem`, the ID is generated by qcontainer, users cannot take for granted what it is.
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qcontainer.py#L345-L377

Reply:
Updated now, thanks!